### ISSUE TITLE
Black bar instead of title bar on Intel Graphics when you have a QOpenGLWidget

### ISSUE THREAD

**Issue by vtronko** [Bringer-of-Light/Qt-Nice-Frameless-Window#36]
![frameless-bug](https://user-images.githubusercontent.com/8950786/64766798-4b104180-d54f-11e9-8394-5ba20b610040.png)

Black rectangle is just 50x50 QOpenGLWidget.

**Comment by vtronko** [comment]
What's more interesting, hitboxes are correct (it means it receives clicks exactly where it should) while painting is done with offset.

It's related to https://bugreports.qt.io/browse/QTBUG-40485?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&showAll=true

Is there any adhoc solution to workaround this?

There's an answer in closed https://github.com/Bringer-of-Light/Qt-Nice-Frameless-Window/issues/19 thread but it's not clear.

**Comment by JosephCoppola-FW** [comment]
I've been suffering from this as well. Any luck on your end on finding a solution @vtronko? 

I've noticed resizing the window after this issue occurs fixes the offset. So I've been trying to figure out if I can trigger a windows event that forces a repaint or something. 

What is very odd is I only see this if I move my app across monitors with the same level of text scaling. If they don't have the same level of text scaling, the offset does not occur.

**Comment by vtronko** [comment]
@JosephCoppola-FW no, I just implemented a fallback version for unsupported chipsets -_- Wasted too much time on this one

**Comment by JosephCoppola-FW** [comment]
Yeah this one is killing us a bit.

**Comment by MisterPascal** [comment]
Hei, is there any solution to this bug? How do you manage it?

**Comment by MisterPascal** [comment]
If anyone comes here again - you can find the bug and a solution here: https://bugreports.qt.io/browse/QTBUG-90701

**Comment by 454461476** [comment]
Has this problem been finally resolved?
